configuration/log-config-0.yaml

```diff
@@ -0,0 +1,81 @@
+# global filter; messages must have at least this severity to pass:
+minSeverity: Debug
+
+# global file rotation settings:
+rotation:
+  rpLogLimitBytes: 5000000
+  rpKeepFilesNum:  10
+  rpMaxAgeHours:   24
+
+# these backends are initialized:
+setupBackends:
+  - KatipBK
+  - EKGViewBK
+  - EditorBK
+  # - AggregationBK
+
+# if not indicated otherwise, then messages are passed to these backends:
+defaultBackends:
+  - KatipBK
+
+# if wanted, the GUI is listening on this port:
+hasGUI: 12870
+
+# if wanted, the EKG interface is listening on this port:
+hasEKG: 12780
```

Comment:
Can we add Prometheus as the default in the demo as well?

Reply:
Prometheus listens on http://localhost:12760/metrics